app/server/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "lint": "eslint . --cache",
     "prebuild": "yarn && rm -rf build",
-    "build": "babel-node ../node_modules/.bin/webpack",
+    "build": "babel-node --presets es2015 $(npm bin)/webpack",
```

Comment:
There is and was always... `.babelrc` is quite unpredictable, because it gets inherited and combined all the way up the directory tree, if I remember correctly, you can't opt out of using it, so having two different configs for babel and webpack babel for instance is impossible. ...I avoid it like the plague.